destination/config/config.go

```diff
@@ -17,15 +17,96 @@
 package config
 
 import (
+	"bytes"
+	"fmt"
+	"strings"
+	"text/template"
+
+	"github.com/Masterminds/sprig/v3"
 	"github.com/conduitio-labs/conduit-connector-redshift/common"
+	"github.com/conduitio/conduit-commons/opencdc"
 )
 
+type TableFn func(opencdc.Record) (string, error)
+
 // Config is a destination configuration needed to connect to Redshift database.
 type Config struct {
 	common.Configuration
+	// Table is the configuration of the table name.
+	Table string `json:"table" default:"{{ index .Metadata \"opencdc.collection\" }}"`
+	// KeyColumns is the configuration of comma-separated column names to build the record key.
+	KeyColumns []string `json:"keyColumns"`
+}
+
+// Init sets lowercase "table" name if not a template.
+func (c Config) Init() Config {
+	if !c.isTableTemplate() {
+		c.Table = strings.ToLower(c.Table)
```

Comment:
Sorry for the last minute comment, I forgot to ask about this: are table names in Redshift case-sensitive?

Reply:
No, table names are not case-sensitive in Redshift.